src/main/java/core/basesyntax/figure/interfaces/DrawFigure.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.figure.interfaces;
+
+public interface DrawFigure {

```

Comment:
```suggestion
public interface Drawable {

```

Reply:
Why don't you write about such errors in names right away?